service/interpreter/activityImpl.go

```diff
@@ -224,12 +225,42 @@ func checkCommandRequestFromWaitUntilResponse(resp *iwfidl.WorkflowStateStartRes
 					return err
 				}
 			}
+			// Check if each command in the combinations has a matching command in one of the lists
+			if !areAllCommandCombinationsIdsValid(commandReq) {
+				return fmt.Errorf("ANY_COMMAND_COMBINATION_COMPLETED can only be used when every command has an commandId that is found in TimerCommands, SignalCommands or InterStateChannelCommands")
```

Comment:
`InterStateChannelCommands` is now renamed to `InternalChannelCommand`. There are still many places in server that hasn't updated. But all the API and error message that it returned should be using the new name for consistency.